src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -46,12 +47,24 @@ public ModelManager(
         this.activityBook = new ActivityBook(activityBook);
         this.internalState = new InternalState(internalState);
         filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        context = new Context();
     }
 
     public ModelManager() {
         this(new AddressBook(), new UserPrefs(), new InternalState(), new ActivityBook());
```

Comment:
do we need to have new Context()?

Reply:
I believe the reasoning is that without a new context, whenever a command tries to check the context you'll get `NullPointerException`

Reply:
as in in the public ModelManager() { ...... } there is no creation of new Context(), is it supposed to be this way?

Reply:
it calls the other constructor that will create a new context so i think it's fine?